package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -606,15 +608,32 @@ def _read_next_timestep(self):
 
         coord_cols = convention_to_col_ix[self.lammps_coordinate_convention]
 
+        # pass possible additional fields from LAMMPS dump
+        # pass velocities
+        velocity_col_names = ["vx", "vy", "vz"]
+        if any(item in attrs for item in velocity_col_names):
+            for cv_name, cv_col_names in enumerate(velocity_col_names):
+                try:
+                    velocity_cols = [attr_to_col_ix[x] for x in velocity_col_names]
+                    ts.has_velocities = True
```

Comment:
This works, but I think we only want to set `ts.has_velocities = True` on exit from this loop, slightly cleaner IMO.